client/app/account/billing/mean/add/billing-mean-add.html

```diff
@@ -160,12 +160,6 @@
                 </button>
             </div>
 
-            <div data-ng-if="mean.validation">
```

Comment:
It seems that both keys doesn't exist anymore:

- `paymentMean_validation_message_documentToSend`
- `paymentMean_validation_message_simpleValidation `